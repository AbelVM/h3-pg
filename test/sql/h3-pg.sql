SELECT h3_haversine_distance('8f2830828052d25', '8f283082a30e623');
SELECT h3_basecells();