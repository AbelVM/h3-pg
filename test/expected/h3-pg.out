\pset tuples_only on
-- neighbouring indexes (one hexagon, one pentagon) at resolution 3
\set hexagon '\'831c02fffffffff\'::h3index'
\set pentagon '\'831c00fffffffff\'::h3index'
\set distance 86.3352698485577
\set epsilon 0.0000000000001
--
-- TEST h3_haversine_distance
--
SELECT h3_haversine_distance(:hexagon, :pentagon) - :distance < :epsilon;
 t

--
-- TEST h3_basecells
--
SELECT COUNT(*) = 122 FROM (SELECT h3_basecells()) q;
 t

